wolfssl-3.9.6/README.md

```diff
@@ -0,0 +1,139 @@
+Finds a SEGV and three heap overflow errors in `src/ssl.c`
+in [wolfSSL v.3.9.6](https://www.wolfssl.com/wolfSSL/Home.html).
```

Comment:
Updated Readme to reflect times. Unfortunately some of the bugs are shallower and thus require fixes for deeper bugs to occur.